src/watchman_watcher.js

```diff
@@ -69,16 +69,18 @@ WatchmanWatcher.prototype.init = function() {
     return self.watchProjectInfo ? self.watchProjectInfo.root : self.root;
   }
 
-  function onVersion(error, resp) {
+  function onCapability(error, resp) {
     if (handleError(self, error)) {
+      // The Watchman watcher is unusable on this system, we cannot continue
+      process.exit(1);
```

Comment:
What would be nicer is if sane could check this before it settles on loading the watchman watcher, and then fall back to one of the other watchers if it isn't present.